### ISSUE TITLE
StatsCAN v3.0.11: Accessibility label for Publications

### ISSUE THREAD

**Issue by weglmar** [StatCan/StatcanPublicationUAT#560]
On the bottom tab bar, Publications is reading as "Saved" or "Enregistre" and 3 of 4 in English and French. 
The label should be modified to Publications (in both languages). and should read 1 of 4.

**Comment by weglmar** [comment]
Seems to be resolved.
@Rajat-Sen, could you please confirm?

**Comment by Rajat-Sen** [comment]
Fixed.
